docs/config.json

```
@@ -416,6 +416,11 @@
               "slug": "/access-controls/access-request-plugins/ssh-approval-pagerduty/",
               "forScopes": ["enterprise", "cloud"]
             },
+            {
+              "title": "Opsgenie",
+              "slug": "/access-controls/access-request-plugins/ssh-approval-opsgenie/",
```

Nit: I understand you made it consistent with other links but I don't think "ssh-approval" makes much sense tbh. I'd rename this to just "opsgenie".
```suggestion
              "slug": "/access-controls/access-request-plugins/opsgenie/",
```